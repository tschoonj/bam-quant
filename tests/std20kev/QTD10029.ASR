File: C:\xrfdaten\2014KW30\chris\std20kev\std10029.spe
Start: 21.07.2014 18:12:57
Stop: 21.07.2014 18:14:37
BESSY: 298.7079997 298.6878172
IONI: 3.361649E-11 3.23535E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 74911
Run: 100000
Dead: 25
ICR: 84220
OCR: 0
Accumulated BESSY: 59703.4935966
Accumulated Ioni: 0.000000006589158
N Accumulation: 200
Normfactor:  4.0518605e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Cu
exp_x 111.260398864746 111.2604
exp_y 122.323806762695 122.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  29   1      8.041   5491307.      2343.     178.49
