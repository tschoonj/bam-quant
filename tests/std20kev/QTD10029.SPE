$DATA:
0 2000
17715
11484
5063
1585
340
58
15
9
4
4
3
0
3
1
3
6
148
1236
1659
1634
1502
1465
1363
1258
1267
1291
1224
1159
1147
1163
1068
1066
1063
1027
987
954
947
948
947
952
866
852
866
830
817
854
788
831
745
772
759
741
739
738
724
783
709
774
724
749
748
723
709
701
692
681
649
660
674
675
662
680
644
626
670
654
630
631
632
590
654
639
606
587
563
622
596
588
643
606
569
641
572
579
603
621
641
609
639
564
641
621
617
672
657
628
589
625
587
594
543
569
587
558
582
540
588
555
549
568
511
514
549
543
558
537
616
562
554
572
587
518
550
556
539
554
492
488
509
516
551
588
595
575
622
636
711
664
666
597
575
548
531
561
501
517
536
525
548
565
535
504
479
492
543
524
496
500
520
467
496
460
494
475
512
506
500
475
482
489
482
519
526
514
519
486
513
530
512
506
463
514
496
522
528
474
515
487
522
472
476
510
471
501
488
477
460
485
514
478
479
545
484
527
507
489
494
509
522
517
525
512
543
541
531
502
526
524
485
554
502
462
554
507
488
547
500
512
495
538
518
521
500
513
519
507
531
513
486
535
507
553
539
545
542
511
545
562
538
540
540
578
536
561
573
539
608
574
543
602
599
539
559
557
588
538
601
596
547
592
613
625
560
584
616
592
626
635
639
605
670
614
636
643
644
678
693
672
663
706
714
724
725
759
816
918
1013
1240
1403
1752
1964
2075
2159
2087
1794
1682
1428
1178
1140
1054
916
867
852
863
806
793
816
873
846
877
916
901
855
880
913
930
952
922
947
953
1054
960
984
1039
1071
1026
1108
1084
1122
1134
1174
1117
1210
1233
1257
1191
1234
1255
1174
1268
1261
1268
1222
1239
1324
1294
1365
1355
1467
1620
1665
1656
1769
1857
1919
2135
2164
2357
2438
2682
2763
3064
3205
3621
4050
4633
5858
8184
13057
22855
42577
76568
130937
210246
313101
426823
534541
614473
645579
620376
546552
441449
325808
221525
137350
79202
42075
21100
10439
5328
3075
2199
1878
1591
1445
1423
1388
1397
1342
1374
1447
1512
1632
1705
1809
1955
2127
2404
2738
3140
3936
5140
7624
11374
17870
27705
40375
55837
72137
86358
95091
97606
92038
80800
64387
47714
32478
20588
12196
6728
3521
1658
804
391
193
135
98
76
66
72
53
67
53
53
58
48
28
37
36
31
32
35
29
37
32
40
26
23
18
34
16
31
26
23
28
20
21
23
24
20
20
24
19
15
27
19
21
21
26
13
24
17
25
27
22
22
27
17
17
17
17
11
19
22
24
26
24
21
26
16
32
33
24
32
25
19
28
19
14
12
27
17
14
16
14
14
14
11
11
14
20
9
13
15
13
13
11
22
7
14
12
13
13
6
16
19
14
12
11
17
11
8
13
17
18
15
10
13
18
15
9
9
11
10
8
13
13
16
7
9
12
8
13
8
13
13
4
5
17
5
9
16
7
6
15
6
3
7
15
6
11
8
7
6
7
11
10
11
9
18
16
10
12
3
9
16
7
11
16
8
23
24
25
19
25
20
27
17
13
28
8
17
12
12
9
6
7
9
11
16
15
14
12
14
11
10
7
14
12
9
12
16
13
9
7
12
7
18
10
9
10
13
13
10
8
5
8
15
7
16
8
12
6
11
10
14
15
11
14
11
13
6
7
8
10
8
7
8
10
12
7
8
11
8
10
11
12
10
5
16
11
11
19
16
16
26
27
22
33
25
27
36
23
12
16
15
15
13
22
16
8
9
8
18
13
14
14
11
12
14
18
12
22
16
16
15
20
11
16
16
17
15
16
10
20
11
19
13
23
16
18
23
17
19
22
19
17
20
17
12
27
16
19
19
15
15
21
22
30
32
28
36
37
38
47
57
47
55
82
75
82
95
109
151
240
296
439
647
971
1224
1727
2050
2369
2812
3022
3128
3161
3015
2641
2352
1992
1613
1212
877
588
427
278
173
93
77
43
32
22
18
28
28
20
27
28
29
40
30
35
56
44
70
95
106
135
202
268
335
461
569
716
773
883
934
921
979
854
816
705
569
448
339
216
182
108
72
50
32
16
8
17
7
4
9
5
5
5
7
4
2
8
6
11
10
8
18
15
21
26
48
54
45
66
60
71
75
91
78
82
64
49
45
48
37
33
15
20
10
6
6
6
2
2
3
1
1
4
4
6
5
4
8
8
2
2
7
1
0
3
5
6
6
5
8
8
7
5
11
10
11
9
7
9
11
7
8
11
12
12
12
16
9
11
10
14
13
13
16
13
11
18
17
21
20
25
18
26
23
25
12
18
24
21
22
18
31
22
26
29
25
35
37
27
30
38
34
37
34
48
38
47
38
48
67
49
65
80
77
84
111
107
107
87
112
104
114
102
123
85
109
112
83
74
91
74
59
42
45
35
31
28
18
17
14
10
9
6
5
7
7
4
5
0
1
1
0
3
2
0
1
0
0
1
0
0
0
0
1
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
1
0
0
1
0
0
0
0
1
0
0
0
0
0
1
0
0
0
0
1
1
0
0
0
0
0
0
0
0
1
0
1
1
1
0
0
0
0
0
0
0
3
2
0
0
3
3
1
2
2
13
4
12
11
9
11
12
14
4
11
7
7
7
6
3
8
3
3
3
1
0
0
0
0
0
0
0
0
1
0
1
0
0
3
0
1
2
0
0
0
2
2
2
4
3
3
8
4
4
6
3
9
5
5
2
2
3
4
0
1
2
0
0
1
0
0
0
1
1
0
0
0
0
0
0
0
0
1
0
0
0
0
3
0
0
0
0
1
0
1
0
1
1
1
1
1
1
1
1
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
1
1
1
1
2
1
0
0
0
1
0
0
1
0
1
0
1
0
0
1
2
0
1
3
3
0
3
0
2
2
0
1
2
2
1
1
3
2
2
3
1
4
2
1
2
2
1
2
0
0
0
0
0
0
2
1
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
1
1
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10029.spe
Start: 21.07.2014 18:12:57
Stop: 21.07.2014 18:14:37
BESSY: 298.7079997 298.6878172
IONI: 3.361649E-11 3.23535E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 74911
Run: 100000
Dead: 25
ICR: 84220
OCR: 0
Accumulated BESSY: 59703.4935966
Accumulated Ioni: 0.000000006589158
N Accumulation: 200
Normfactor:  4.0518605e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Cu
 
exp_x 111.260398864746 111.2604
exp_y 122.323806762695 122.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  29   1      8.041   5491307.      2343.     178.49
