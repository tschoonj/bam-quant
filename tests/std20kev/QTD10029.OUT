     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:43  บ   
     บ  Spectrum: QTD10029.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        4 iterations done    บ   
     บ  ChiSquare =   142.8   last change =   .01%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -39.4 ๑   100.0   บ     -39.4 ๑      .4   บ   
     บ  GAIN (eV/ch)  บ     20.251 ๑   2.000   บ    20.251 ๑    .001   บ   
     บ det NOISE (eV) บ      -81.1 ๑    40.0   บ     -81.1 ๑    -2.7   บ   
     บ  FANO factor   บ       .193 ๑    .050   บ      .193 ๑    .002   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Cu-Ka                           5491307. ๑   2343.           บ   
     บ    KA1         8.048    .66084     3628892. ๑   1549.    178.5  บ   
     บ              399.365    163.31         241.           9.57E-01  บ   
     บ    KA2         8.028    .33916     1862415. ๑    795.    162.5  บ   
     บ              398.377    163.06         242.           9.57E-01  บ   
     บ                                                                 บ   
     บ  2 Cu-Kb                            882152. ๑    940.           บ   
     บ    KB1         8.905   1.00000      882152. ๑    940.     69.4  บ   
     บ              441.684    173.81         244.           9.68E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                           50236. ๑    225.           บ   
     บ    L1N2       16.082    .74230       37290. ๑    167.     18.9  บ   
     บ              796.113    244.66          91.           9.87E-01  บ   
     บ    L1O3       16.946    .23854       11983. ๑     54.      4.0  บ   
     บ              838.767    251.84          74.           9.85E-01  บ   
     บ    L1O2       17.810    .01916         963. ๑      9.      1.6  บ   
     บ              881.422    258.83          58.           9.80E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
